src/main/java/SasDefinitionSample.java

```diff
@@ -1,230 +1,215 @@
+                        accountSasPermission,
+                        accountSasService,
+                        accountSasResourceType);
+        //Generate a signature based off of the policy and account.
+        com.azure.storage.blob.BlobServiceClient blobServiceClient
+                = new com.azure.storage.blob.BlobServiceClientBuilder()
+                .endpoint(storageAccount.endPoints().primary().blob())
+                .credential(sas).buildClient();
+        String sasSignature = blobServiceClient.generateAccountSas(accountSasSignatureValues);
+        System.out.println("Generated sasSignature " + sasSignature);
+
+        // Generating new cloud storage account object off of the new acctSasToken
+        blobServiceClient = new com.azure.storage.blob.BlobServiceClientBuilder()
+                .endpoint(storageAccount.endPoints().primary().blob())
+                .sasToken(sasSignature).buildClient();
+        BlobContainerClient blobContainerClient
+                = blobServiceClient.getBlobContainerClient("cloudstorageblob");
+        if (!blobContainerClient.exists()) {
+            blobContainerClient.create();
+        }
+
+        // Creating a blob with the account storage token.
+        System.out.println("Creating container: " + blobContainerClient.getBlobContainerName());
+        BlockBlobClient blockBlobClient =blobContainerClient.getBlobClient("blobName").getBlockBlobClient();
+        String text = "test blob data";
+        System.out.printf("Uploading text: \"%s\" to blob%n", text);
+        blockBlobClient.upload(new ByteArrayInputStream(text.getBytes()), text.getBytes().length);
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        blockBlobClient.download(outputStream);
+        System.out.println("Downloading text: " + new String(outputStream.toByteArray(),"UTF-8"));
+        blockBlobClient.delete();
+        System.out.println("Blob deleted");
+    }
+
+    /**
+     * Creates a service SAS definition with access to a blob container.
+     */
+    public void createBlobSasDefinition() throws UnsupportedEncodingException {
+
+        // Create the blob sas definition template
+        // The sas template uri for service sas definitions contains the storage entity uri with the template token
+        // This sample demonstrates constructing the template uri for a blob container, but a similar approach can
+        // be used for all other storage service, i.e. File, Queue, table
+
+        // Create a template sas token for a container
+        StorageSharedKeyCredential sas = new StorageSharedKeyCredential(STORAGE_ACCOUNT_NAME, storageAccount.getKeys().get(0).value());
+        // Note that the key passed in is just a dummy key such that we can generate the correct signature for the template.
+
+        com.azure.storage.blob.BlobServiceClient blobServiceClient
+                = new com.azure.storage.blob.BlobServiceClientBuilder()
```

Comment:
Let's import this instead of using the fully qualified name.
```suggestion
                = new BlobServiceClientBuilder()
```